packages/sdk-core/package.json

```diff
@@ -9,18 +9,27 @@
     "module": "dist/module/index.js",
     "types": "dist/main/index.d.ts",
     "files": [
-        "dist/**/*.js",
-        "dist/**/*.d.ts"
+        "dist/main",
+        "dist/module",
+        "dist/docs",
+        "dist/index.umd.js",
+        "!**/*.json",
+        "CHANGELOG.md",
+        "LICENSE",
+        "README.md"
```

Comment:
I unified this section with `sdk-redux`. Looks OK, right?

Reply:
yeah